Jenkinsfile

```diff
@@ -30,18 +29,72 @@ pipeline {
 		stage('Setup: Back up DB'){
 			steps{
 				script{
-          withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-            def release_current_before_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_before_biomodels.dump"
-            sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_before_biomodels_dump"
-            sh "gzip -f $release_current_before_biomodels_dump"
-          }
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def release_current_before_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_before_biomodels.dump"
+						sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_before_biomodels_dump"
+						sh "gzip -f $release_current_before_biomodels_dump"
+          				}
+				}
+			}
+		}
+		stage('Setup: Create reactome database from release_current') {
+			steps{
+				script{
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]) {
+						sh "mysql -u$user -p$pass -e \'drop database if exists ${env.REACTOME}; create database ${env.REACTOME}\'"
+						sh "zcat ${env.RELEASE_CURRENT}_${currentRelease}_before_biomodels.dump.gz | mysql -u$user -p$pass ${env.REACTOME}"
+					}
+				}
+			}
+		}
+		// This stage generates the graph database using the graph-importer module, and replaces the current graph db with it.
+		stage('Setup: Generate Graph Database'){
+			steps{
+				script{
+					cloneOrPullGitRepo("release-jenkins-utils")
+					sh "cp -f release-jenkins-utils/scripts/changeGraphDatabase.sh ${env.JENKINS_HOME_PATH}"
+					sh "chmod 700 ${env.JENKINS_HOME_PATH}changeGraphDatabase.sh"
```

Comment:
Similar to https://github.com/reactome/release-jenkins-utils/pull/3#discussion_r436122019 - including the slash separator in the paths may be easier to read than in the variable